From 8235d0ccdbe6acbf3ae48e74bf83cf48d8dae08b Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Mon, 26 May 2025 11:53:13 +0100
Subject: [PATCH 1/1] data: Add "Categories" to .desktop files

Although specifying a category isn't required by the "Desktop Entry
Specification"; `appstreamcli compose` will error if a registered
category isn't specified.

Upstream issue: https://github.com/ximion/appstream/issues/577

Fixes composing rofi into the catalog for distros that use
`appstream-generator`.
---
 data/rofi-theme-selector.desktop | 1 +
 data/rofi.desktop                | 1 +
 2 files changed, 2 insertions(+)

diff --git a/data/rofi-theme-selector.desktop b/data/rofi-theme-selector.desktop
index 6f3b8e07..21a32a14 100644
--- a/data/rofi-theme-selector.desktop
+++ b/data/rofi-theme-selector.desktop
@@ -6,3 +6,4 @@ Terminal=false
 Exec=rofi-theme-selector
 Name=Rofi Theme Selector
 Icon=rofi
+Categories=Utility
diff --git a/data/rofi.desktop b/data/rofi.desktop
index 74a03484..e1ec1e2a 100644
--- a/data/rofi.desktop
+++ b/data/rofi.desktop
@@ -6,3 +6,4 @@ Terminal=false
 Exec=rofi -show
 Name=Rofi
 Icon=rofi
+Categories=Utility
-- 
2.49.0

